data/db/src/commonMain/sqldelight/io/github/droidkaigi/feeder/data/TimetableItemSession.sq

```diff
@@ -3,7 +3,19 @@ CREATE TABLE timetableItemSession (
     jaTitle TEXT NOT NULL,
     enTitle TEXT NOT NULL,
     startsAt INTEGER NOT NULL,
-    endsAt INTEGER NOT NULL
+    endsAt INTEGER NOT NULL,
```

Comment:
Can you add `3.sqm` empty file in src/commonMain/sqldelight/io/github/droidkaigi/feeder/data like 2.sqm?
It is needed for the migration 🙏

Reply:
I added the file. 1293754